.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+
+  - package-ecosystem: "docker"
+    directory: "/"
```

Comment:
I'm not sure this will work as expected because while the `docker-compose.yml` is in the root of the directory, our Dockerfiles will soon be moved to the [`docker/` directory](https://github.com/creativecommons/vocabulary/tree/master/docker) to match the folder structure of Vocabulary.

Reply:
@dhruvkb Ah okay. I will send a PR to cc-vocabulary first then.